docs/sphinx/user_guide/run.rst

```diff
@@ -179,16 +202,15 @@ starting weights (sum to 1.0) which include:
 
 .. note:: Caveats: 
 
-          #. When collecting PAPI data in this way you'll be limited to running              only one variant, since Caliper maintains only one PAPI context.
+          #. When collecting PAPI data in this way you'll be limited to running              only one kernel variant, since Caliper maintains only one PAPI
```

Comment:
```suggestion
          #. When collecting PAPI data in this way you'll be limited to running
             only one kernel variant, since Caliper maintains only one PAPI
```